source/IntegrationTests/Behaviours/IncomingRequests/GivenMeteredDataForMeteringPointV2Tests.cs

```diff
@@ -193,4 +207,127 @@ await ThenNotifyValidatedMeasureDataDocumentIsCorrect(
                                     Product: "8716867000030")),
                         ])));
     }
+
+    [Theory]
+    [MemberData(nameof(SupportedRejectDocumentFormats))]
+    public async Task AndGiven_InvalidBusinessReason_When_ActorPeeksMessage_Then_ReceivesOneRejectDocumentWithCorrectContent(DocumentFormat documentFormat)
+    {
+        /*
+         *  --- PART 1: Receive request and send message to Process Manager ---
+         */
+
+        // Arrange
+        var senderSpyStart = CreateServiceBusSenderSpy(StartSenderClientNames.Brs021ForwardMeteredDataStartSender);
+        var senderSpyNotify = CreateServiceBusSenderSpy(NotifySenderClientNames.Brs021ForwardMeteredDataNotifySender);
+        var senderActor = (ActorNumber: ActorNumber.Create("1111111111111"), ActorRole: ActorRole.GridAccessProvider);
+        var orchestrationInstanceId = Guid.NewGuid();
+        var resolution = Resolution.Hourly;
+        var notifyEventName = ForwardMeteredDataNotifyEventV1.OrchestrationInstanceEventName;
+        var messageId = MessageId.New();
+
+        var registeredAt = Instant.FromUtc(2022, 12, 17, 9, 30, 00);
+        var startDate = Instant.FromUtc(2024, 11, 28, 13, 51);
+        var endDate = Instant.FromUtc(2024, 11, 29, 9, 15);
+
+        var expectedEnergyObservations = new List<(int Position, string? QualityName, decimal? Quantity)>
+        {
+            (1, null, null),
+            (2, "A03", null),
+            (3, null, 123.456m),
+            (4, "A03", 654.321m),
+        };
+
+        var now = Instant.FromUtc(2024, 7, 1, 14, 57, 09);
```

Comment:
```suggestion
        var now = Instant.FromUtc(2025, 7, 1, 14, 57, 09);
```
Needs to be after "enddate"?